app/views/account/deletions/show.html.erb

```diff
@@ -31,5 +31,13 @@
   <li><%= t ".retain_email" %></li>
 </ul>
 
-<%= link_to t(".delete_account"), account_path, { :method => :delete, :class => "btn btn-danger", :data => { :confirm => t(".confirm_delete") } } %>
+<% if @allowed_at %>
+  <div class="alert alert-warning">
+    <%= t ".recent_editing_html", :time => friendly_date_time_element(@allowed_at) %>
+  </div>
+  <button class="btn btn-danger" disabled><%= t(".delete_account") %></button>
```

Comment:
I wonder if we should just leave the button out when deletion is not possible? Even a disabled "danger" button still looks very clickable - the difference to a non-disabled one is quite subtle.

Reply:
On my mockup I used btn-secondary instead of btn-danger along with the disabled attribute.

Reply:
changed to btn-secondary